src/pages/auth/type.ts

```diff
@@ -1,6 +1,24 @@
+import { ButtonProps } from '@chakra-ui/react'
+
 export enum ExternalAccountType {
   nostr = 'nostr',
   twitter = 'twitter',
   lightning = 'lnurl',
   fountain = 'Fountain',
+  facebook = 'facebook',
+  google = 'google',
+  github = 'github',
+}
+
+export enum SocialAccountType {
+  facebook = 'facebook',
+  google = 'google',
```

Comment:
Do we count google as a social account?

Reply:
Yes, because it uses the same flow as others, and honestly we get more information from Google than facebook. The user's profile image and username would be set if they log in with google.